model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/entities/CachedRealm.java

```diff
@@ -249,10 +245,6 @@ public CachedRealm(Long revision, RealmModel model) {
         requiredCredentials = model.getRequiredCredentialsStream().collect(Collectors.toList());
         userActionTokenLifespans = Collections.unmodifiableMap(new HashMap<>(model.getUserActionTokenLifespans()));
 
-        this.identityProviders = model.getIdentityProvidersStream().map(IdentityProviderModel::new)
-                .collect(Collectors.toList());
-        this.identityProviders = Collections.unmodifiableList(this.identityProviders);
-
         this.identityProviderMapperSet = model.getIdentityProviderMappersStream().collect(Collectors.toSet());
```

Comment:
It reminds me that we'll need to sort mappers in a follow-up. Having a single mapper on each broker won't scale as well.

Reply:
Yes, absolutely! I think we should focus on that right in the sequence of this work